test/factree/pathfinder_test.rb

```diff
@@ -0,0 +1,60 @@
+require 'test_helper'
+
+describe Factree::Pathfinder do
+  describe "#find_node_sequence" do
+    let(:facts) do
```

Comment:
We should move away from `let` since we are using Minitest. `let` was begrudgingly added to Minitest::Spec to help people coming over from RSpec. It's more idiomatic and less confusing to use standard Ruby methods and/or ivars.